platform-tests/src/test/java/org/junit/AutomaticModuleNameTests.java

```diff
@@ -61,10 +61,10 @@ private static String version(String module) {
 
 	static Stream<String> moduleDirectoryNames() throws IOException {
 		// @formatter:off
-		return Files.walk(Paths.get(".."), 1)
-				.filter(Files::isDirectory)
-				.map(Path::getFileName)
-				.map(Object::toString)
+		String startOfModuleLine = "include '";
+		return Files.readAllLines(Paths.get("../settings.gradle")).stream()
```

Comment:
Oops, sorry, `File.walk` is for directories, not files. Silly me. :stuck_out_tongue:

Try `Files.lines(Paths.get("../settings.gradle", StandardCharsets.UTF_8))` instead.